Update docker-machine.md

InsideTechnologies opened the pull request:
VM Sizing is mandatory because the script use Standard A2 that is not more available. It's important specify that is necessary use bash also in Windows 10, because docker-machine is present if Docker for Windows is installed.

azurecla commented:
Hi __@hanuk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hanuk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

InsideTechnologies committed:
Update docker-machine.md

VM Sizing is mandatory because the script use Standard A2 that is not more available. It's important specify that is necessary use bash also in Windows 10, because docker-machine is present if Docker for Windows is installed.
